boot/freeldr/freeldr/arch/i386/pc98/pc98video.c

```diff
@@ -336,4 +453,13 @@ Pc98VideoPrepareForReactOS(VOID)
     Int386(0x18, &Regs, &Regs);
 
     Pc98VideoHideShowTextCursor(FALSE);
+
+    if (UseCGFont && CGAccelDraw)
```

Comment:
```suggestion
    /* Clear the screen, resetting to default attributes */
    if (UseCGFont && CGAccelDraw)
```
or something like that :)